### DISCUSSION TITLE
Is the 6th form / Twisted Corner possible with Neovox?

### DISCUSSION THREAD

**Discussion by Kaiser-Indrasil** [tsunamayo/Starship-EVO#3950] [Suggestions]
The Prefab system was an amazing addition to what already is the best spaceship building game on the market.
I've been having a blast with the building system in the Experimental branch with the multi-symmetry capability, fixed inter-entity collision checks, and the Neovox scale tech, of course.
I don't need clipping anymore.
Hell, I don't even need to go back to the Legacy branch anymore.
The building system is pretty much fleshed out.

Maybe with the exception of that one last block.
You know, which one.
The one that was unfeasible to introduce because of mirroring issues.
I'm aware that you've heard us those few times we suggested it in the past, but I'm wondering if it's feasible now that the Neovox tech is in? 🤔

![image](https://user-images.githubusercontent.com/63756057/124767423-b64eb300-df2f-11eb-8712-b7914f0585b1.png)

There's a few reasons behind implementing this particular block:
- lessen the CE usage. Sometimes when I'm building a slanted piece of armour, I need to place it on a hinge to pull off a particular shape, like a ramp on that slanted armour piece. The Twisted Corner block would remedy this.

![image](https://user-images.githubusercontent.com/63756057/124768301-76d49680-df30-11eb-96de-c6237c831588.png)

- support for sliding mechanisms moving against slanted surfaces (thanks to @Pawnp for providing that use case).

![image](https://user-images.githubusercontent.com/63756057/124769201-2f9ad580-df31-11eb-9ab7-175e695e0cef.png)

**Comment by ProPeach** [comment]
Lots of discussion over on the main thread of #3853 too

**Comment by Uncle-Ulty** [comment]
This is the missing form to make leaned/curved wings,fins, edges and so on
